webpack/assets/javascripts/react_app/components/HostDetails/Properties/__snapshots__/Properties.test.js.snap

```diff
@@ -29,7 +29,13 @@ exports[`HostDetails - Interfaces should render UserProfile 1`] = `
                 </span>
               </DataListCell>,
               <DataListCell>
-                windows 3.11
+                <SkeletonLoader
+                  emptyState="N/A"
+                  skeletonProps={Object {}}
+                  status="PENDING"
```

Comment:
Should this status be pending? I see the `isLoading` prop [in fixtures](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/HostDetails/Properties/Properties.test.js#L16) that probably should be changed as `Properties` component now [destructs props](https://github.com/theforeman/foreman/pull/8314/files#diff-af4fca9b0bca4df0e7fb90c42327f482b8af577d1780f7d146572612f77ae9e1R15) to `{hostData, status}`

Reply:
thanks 👍